tests/test_modeling_common.py

```diff
@@ -133,6 +134,7 @@ class ModelTesterMixin:
     all_model_classes = ()
     all_generative_model_classes = ()
     fx_compatible = False
+    fx_trace_can_be_torchscripted = True
```

Comment:
Yes, I see it's a new flag. That's what I would like to avoid.

Reply:
This is just for Swin, so you can override the new test and skip the part inside the swin testing file. My concern is that this gives several flags to learn for **every** contributor of new models, and they won't know the difference between all of them (right now we already have two with fx and torchscript).

Reply:
I don't see how it's bad?

I am testing that traced models can be torchscripted [here](https://github.com/huggingface/transformers/pull/17206/files#diff-a6ebb0bccfe159ca855e9d9b5d6fbbeb62fced0c0296de0993ba39ef5c862cf0R788), without the flag I cannot really know when this should be tested.
Instead of using the flag, I could hardcode the classes to ignore but that seems worse IMO.